app/views/articles/show.html.erb

```diff
@@ -114,7 +114,8 @@
         id="article-show-container"
         data-author-id="<%= @article.user_id %>"
         data-path="<%= @article.path %>"
-        data-published="<%= @article.published? %>">
+        data-published="<%= @article.published? %>"
+        data-discussion-locked="<%= @discussion_lock.present? %>">
```

Comment:
So the JS knows if the discussion is locked or not

Reply:
I think we don't use `@discussion_lock` for anything else, so maybe we can update the controller to 

```ruby
@discussion_locked = article.discussion_lock.present?
```

and then use that here. IMO the less a view knows about outside data and its structure, the better.

Reply:
We also use it in 2 other places:

https://github.com/forem/forem/pull/13808/files#diff-11f5f351c9e82eb6d2903ab4395ece443a1475441d8f26debbe2520c0c8ba385R6

https://github.com/forem/forem/pull/13808/files#diff-11f5f351c9e82eb6d2903ab4395ece443a1475441d8f26debbe2520c0c8ba385R12

Reply:
Ah, sorry, I missed this then, thanks!